modules/fedimint-ln-client/src/lib.rs

```diff
@@ -1185,6 +1185,23 @@ impl LightningClientModule {
         operation_id: OperationId,
     ) -> anyhow::Result<UpdateStreamOrOutcome<InternalPayState>> {
         let operation = self.client_ctx.get_operation(operation_id).await?;
+
+        let LightningOperationMetaVariant::Pay(LightningOperationMetaPay {
+            out_point: _,
+            invoice: _,
+            change: _, // FIXME: why isn't this used here?
```

Comment:
I believe the outpoint for this isn't used because it is passed through the `RefundSubmitted` state.